gui/wxpython/psmap/frame.py

```diff
@@ -258,6 +258,17 @@ def _checkMapFrameExists(self, type_id):
                 return False
         return True
 
+    def _switchToPage(self, current_page_index=1, page_index=0):
+        """Switch to page (default to Draft page)
+
+        :param int current_page_index: current page index from you
+        want switch
+        :param int page_index: page index where you want to switch
+        """
+        if self.currentPage == current_page_index:
```

Comment:
I am not sure what is the point of current_page_index and the if condition? Why not just have one parameter, page_index?

Reply:
Fixed in the [56f29c5](https://github.com/OSGeo/grass/pull/1057/commits/56f29c58d0b3c939ffb2b4f4f0c313553c83e4da).